CMakeLists.txt

```diff
@@ -442,6 +443,26 @@ set (UPM_COMMON_HEADER_DIRS ${CMAKE_HOME_DIRECTORY}/include)
 # Generate a build-only C++ header to add functionality to SWIG'ed modules
 configure_file (${PROJECT_SOURCE_DIR}/cmake/modules/version.hpp.in ${PROJECT_BINARY_DIR}/src/version.hpp @ONLY)
 
+# Cordova binding
+if (BUILDCORDOVABINDING)
+  if (BUILDSWIGJAVA)
+  else()
+    message(FATAL_ERROR "Must enable java binding first")
+  endif()
+
+  find_package (Node REQUIRED)
+  find_package (Npm REQUIRED)
+
+  add_custom_target(cordova_binding ALL
+                    COMMAND ${NPM_EXECUTABLE} install -g upm-cordova-binding
```

Comment:
This is installing a global npm package, which will make this work. But what happens if you already have a upm-cordova-binding already installed? 

I think it could best to just check if the executable is installed, (like we do with Yuidoc for example), and fail with  information on how to install the package, instead of installing it for you

It could be better if you enforce a given version of the npm package here, otherwise it will always try to use the latest version, which could cause problems with backward compatibility

Also, I would like to see some information in the building.md folder on where to get this package, how to install it, etc

Reply:
I will replace the part of installing the npm package with just checking the existence of it. But I will keep it tracking latest version until the package gets a stable release.

Reply:
Thanks! I would also like to know where the code of this npm package lives!